config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -0,0 +1,331 @@
+periodics:
+# Experimental config for testing private gce clusters.
+- name: ci-kubernetes-e2e-gce-private-cluster-correctness
+  # TODO(mm4tt): Lower the timeout once we know how long the test takes.
+  interval: 6h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-common: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190716-41e2a9e-master
+      args:
+      - --timeout=300
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=private-gce-test
+      - --env=KUBE_GCE_PRIVATE_CLUSTER=true
+      - --extract=ci/latest
+      - --gcp-master-image=gci
+      - --gcp-node-image=gci
+      - --gcp-nodes=5
+      - --gcp-ssh-proxy-instance-name=private-gce-test-master
+      - --gcp-zone=us-east1-b
+      - --ginkgo-parallel=5
+      - --provider=gce
+      - --test_args=--ginkgo.flakeAttempts=2 --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[DisabledForLargeClusters\] --minStartupPods=8 --node-schedulable-timeout=90m
+      - --timeout=240m
+      - --use-logexporter
+  annotations:
```

Comment:
Actually, I just sent out a PR to unify stuff a bit in our configs: https://github.com/kubernetes/test-infra/pull/13516
So if you could move annotations right after labels and add an empty line between configs, it would be great.

Reply:
fixed, thanks!